proto/gcpb.proto

```diff
@@ -111,16 +111,16 @@ message UpdateServiceSafePointByServiceGroupResponse {
   uint64 new_safe_point = 5;
 }
 
-message ServiceGroupSafePoint {
-  bytes service_group_id = 1;
+message KeySpaceSafePoint {
```

Comment:
This safepoint is gc safepoint? Is't better to be named  ```KeySpaceGCSafePoint``` ？

Reply:
ok, that make sense